source/reference/mongosync.txt

```diff
@@ -296,6 +296,24 @@ error occurs, the ``mongosync`` log may contain the word "error' but
 ``mongosync`` is still able to complete the sync. In the case that a
 sync does not complete, ``mongosync`` writes a fatal log entry. 
 
+UUIDs
+~~~~~
+
+``mongosync`` creates new universally unique identifiers (UUIDs) for
```

Comment:
I don't know if we should use this:
:abbr:\`UUIDs (universally unique identifiers)\`

Reply:
Done! (I was not aware of this markup so thank you so much for pointing it out!)